docs-ref-services/preview/communication-administration-readme.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Communication Administration client library for Java
+description: Describes the Azure Communication Administration client library and provides prerequisites, key concepts, and examples.
 keywords: Azure, java, SDK, API, azure-communication-administration, communication
 author: ramya-rao-a
-ms.author: ramyar
+ms.author: sizhu
 ms.date: 11/17/2020
 ms.topic: reference
-ms.prod: azure
-ms.technology: azure
 ms.devlang: java
-ms.service: communication
+ms.service: azure
```

Comment:
What is the problem of the ms.service value?

Reply:
The 'communication' ms.service value was considered an invalid value.